### ISSUE TITLE
Using the latest generated build of iris (iris-mc1.18.2-1.2.5-pre-619ad432) causes vanilla tweaks bushy leaves to bug out

### ISSUE THREAD

**Issue by Funtimes909** [IrisShaders/Iris#1447]
### What happened?

Title, When using this build of iris major graphical errors happen to Vanilla Tweaks bushy leaves and act as if graphics is set to fast
This is using Sildurs Enhanced Default Shaders (Version 1.11) No other shader packs tested
Does not occur on iris version 1.2.4

### Screenshots

![Uploading image.png…]()

### Relevant log output

```shell
10:27:29.058
Iris
Render thread
Profile: Fast (+0 options changed by user)
10:27:29.102
Iris
Render thread
Using shaderpack: Sildurs_Enhanced_Default_v1.11.zip
10:27:29.103
Iris
Render thread
Creating pipeline for dimension OVERWORLD
10:27:29.297
ChunkBuilder
Render thread
Stopping worker threads
10:27:29.314
ChunkBuilder
Render thread
Started 12 worker threads
10:27:29.337
net.minecraft.class_338
Render thread
[CHAT] Toggled shaders to Sildurs_Enhanced_Default_v1.11.zip!
10:29:54.481
net.minecraft.server.MinecraftServer
Server thread
10:29:54.483
net.minecraft.class_338
Render thread
[CHAT] Set the time to 1000
10:29:57.684
net.minecraft.class_1132
Server thread
Saving and pausing game...
10:29:57.697
net.minecraft.server.MinecraftServer
Server thread
Saving chunks for level 'ServerLevel[seilurtbditgbklxfsthsvaipdortan]'/minecraft:overworld
10:29:58.970
net.minecraft.server.MinecraftServer
Server thread
Saving chunks for level 'ServerLevel[seilurtbditgbklxfsthsvaipdortan]'/minecraft:the_end
10:29:58.971
net.minecraft.server.MinecraftServer
Server thread
Saving chunks for level 'ServerLevel[seilurtbditgbklxfsthsvaipdortan]'/minecraft:the_nether
10:30:04.983
net.minecraft.class_1132
Server thread
Saving and pausing game...
10:30:05.001
net.minecraft.server.MinecraftServer
Server thread
Saving chunks for level 'ServerLevel[seilurtbditgbklxfsthsvaipdortan]'/minecraft:overworld
10:30:05.041
net.minecraft.server.MinecraftServer
Server thread
Saving chunks for level 'ServerLevel[seilurtbditgbklxfsthsvaipdortan]'/minecraft:the_end
10:30:05.041
net.minecraft.server.MinecraftServer
Server thread
Saving chunks for level 'ServerLevel[seilurtbditgbklxfsthsvaipdortan]'/minecraft:the_nether
10:34:14.634
net.minecraft.class_1132
Server thread
Saving and pausing game...
10:34:14.639
net.minecraft.server.MinecraftServer
Server thread
Saving chunks for level 'ServerLevel[seilurtbditgbklxfsthsvaipdortan]'/minecraft:overworld
10:34:14.658
net.minecraft.server.MinecraftServer
Server thread
Saving chunks for level 'ServerLevel[seilurtbditgbklxfsthsvaipdortan]'/minecraft:the_end
10:34:14.659
net.minecraft.server.MinecraftServer
Server thread
Saving chunks for level 'ServerLevel[seilurtbditgbklxfsthsvaipdortan]'/minecraft:the_nether
10:34:16.897
Iris
Render thread
Destroying pipeline OVERWORLD
10:34:16.912
Iris
Render thread
Shaders are disabled because enableShaders is set to false in iris.properties
10:34:16.912
Iris
Render thread
Shaders are disabled
10:34:16.913
Iris
Render thread
Creating pipeline for dimension OVERWORLD
10:34:16.943
ChunkBuilder
Render thread
Stopping worker threads
10:34:16.948
ChunkBuilder
Render thread
Started 12 worker threads
10:34:16.967
net.minecraft.class_338
Render thread
[CHAT] Shaders disabled!
10:34:17.505
Iris
Render thread
Destroying pipeline OVERWORLD
10:34:17.526
Iris
Render thread
Profile: Fast (+0 options changed by user)
10:34:17.568
Iris
Render thread
Using shaderpack: Sildurs_Enhanced_Default_v1.11.zip
10:34:17.568
Iris
Render thread
Creating pipeline for dimension OVERWORLD
10:34:17.771
ChunkBuilder
Render thread
Stopping worker threads
10:34:17.776
ChunkBuilder
Render thread
Started 12 worker threads
10:34:17.795
net.minecraft.class_338
Render thread
[CHAT] Toggled shaders to Sildurs_Enhanced_Default_v1.11.zip!
10:34:19.034
net.minecraft.class_1132
Server thread
Saving and pausing game...
10:34:19.045
net.minecraft.server.MinecraftServer
Server thread
Saving chunks for level 'ServerLevel[seilurtbditgbklxfsthsvaipdortan]'/minecraft:overworld
10:34:19.065
net.minecraft.server.MinecraftServer
Server thread
Saving chunks for level 'ServerLevel[seilurtbditgbklxfsthsvaipdortan]'/minecraft:the_end
10:34:19.065
net.minecraft.server.MinecraftServer
Server thread
Saving chunks for level 'ServerLevel[seilurtbditgbklxfsthsvaipdortan]'/minecraft:the_nether
10:34:22.033
net.minecraft.class_1132
Server thread
Saving and pausing game...
10:34:22.039
net.minecraft.server.MinecraftServer
Server thread
Saving chunks for level 'ServerLevel[seilurtbditgbklxfsthsvaipdortan]'/minecraft:overworld
10:34:22.050
net.minecraft.server.MinecraftServer
Server thread
Saving chunks for level 'ServerLevel[seilurtbditgbklxfsthsvaipdortan]'/minecraft:the_end
10:34:22.050
net.minecraft.server.MinecraftServer
Server thread
Saving chunks for level 'ServerLevel[seilurtbditgbklxfsthsvaipdortan]'/minecraft:the_nether
```

### Minecraft Version

1.18.2

### Iris Version

iris-mc1.18.2-1.2.5-pre-619ad432

### Sodium Version

sodium-fabric-mc1.18.2-0.4.1+build.15

### Operating System

Windows 10

### What is your GPU?

1660 TI

### Java Version

JDK 17

### Additional context

Moving around and changing angles sometimes causes them to revert back to normal

**Comment by coderbot16** [comment]
It seems like the screenshot did not upload

**Comment by Funtimes909** [comment]
> It seems like the screenshot did not upload

Would you be able to message me on discord where I can see a screenshot there? (Funtimes909#6969) If not I can try and re-upload it here?

**Comment by coderbot16** [comment]
Here is good, it just takes time to upload.

**Comment by Funtimes909** [comment]
![Funtimes909-22-05-2022-10-30-04-javaw](https://user-images.githubusercontent.com/68979259/169676714-f37eeda4-c63c-4965-b4ec-5141167617b0.png)

**Comment by IMS212** [comment]
Should be fixed with #1459.

**Comment by coderbot16** [comment]
Should be fixed in latest 1.2.5 beta 👍
